src/Client.php

```diff
@@ -79,7 +79,7 @@ public function __construct(LoopInterface $loop = null, $url = null)
             throw new \InvalidArgumentException('Invalid Docker Engine API URL given');
         }
 
-        $browser = new Browser($loop, $connector);
+        $browser = new Browser($connector);
```

Comment:
Argument should not be omitted if explicit loop is given.

Reply:
:+1: